include/cmdlist_conv.hpp

```diff
@@ -242,7 +243,11 @@ class CDMPDVCmdListConvHelper : public CDMPDVCmdListKHelper {
 
       kcmd.run[i_run].actfunc = cmd->run[i_run].actfunc;
       kcmd.run[i_run].actfunc_param = cmd->run[i_run].actfunc_param;
-      kcmd.run[i_run].conv_dilation = cmd->run[i_run].conv_dilation;
+      if (cmd->run[i_run].conv_dilation & 0xfefe) {
+        kcmd->run[i_run].conv_dilation = cmd->run[i_run].conv_dilation;
+      } else {
+        kcmd->run[i_run].conv_dilation = 0;
+      }
```

Comment:
May be it is better to use single-line version:
`kcmd->run[i_run].conv_dilation = (cmd->run[i_run].conv_dilation & 0xfefe) ? cmd->run[i_run].conv_dilation : 0;`

Reply:
Done